contrib/initramfs/zfsunlock

```diff
@@ -8,7 +8,7 @@ while [ ! -e /run/zfs_fs_name ]; do
 	if [ -e /run/zfs_unlock_complete ]; then
 		exit 0
 	fi
-	sleep 0.5
+	sleep 1
```

Comment:
This seems to be no longer enforced by newer checkbashisms versions: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=843913 (reported in 2016, fixed in early 2020. Ouch.)
Indeed, checkbashisms on Arch doesn't report this anymore.

As far as I can tell, busybox (tested), GNU (tested) and (at least Free-)BSD ([1]) `/bin/sleep` accept floats.

Sadly there seems to be no way to exclude that check, so I'm unsure what to recommend. I guess half a second more to silence a wrong warning isn't the end of the world, but still not nice to see.

[1] https://www.freebsd.org/cgi/man.cgi?sleep(1)

Reply:
I agree with your reasoning. 

I just found something: Testing on Debian 9 showed moving the float value to a variable seemed to resolve the warning. I don't have time to test other distros/releases -> checkbasisms versions right now.

Otherwise I'd just keep the patch as is and maaaybe take note of this (in a new issue?), to possibly change it back at a later point. But then again, it's half a second more, might not even be worth the effort. 🤷‍♂️

Reply:
That's an interesting workaround.  Let's go ahead and keep the patch is for now.  We can always revisit switching it back to half a second if someone determines it's worth the effort.